oauthproxy.go

```diff
@@ -930,6 +932,14 @@ func (p *OAuthProxy) addHeadersForProxying(rw http.ResponseWriter, req *http.Req
 		}
 	}
 
+	if p.SetBasicAuth {
+		if session.User != "" {
+			rw.Header().Set("Authorization", fmt.Sprintf("Basic %s", b64.StdEncoding.EncodeToString([]byte(session.User+":"+p.BasicAuthPassword))))
```

Comment:
This option will clash with `SetAuthorizationHeader`, can we add some validation to ensure they aren't set simultaneously please?

Reply:
fixed according to the feedback